.github/workflows/ci.yml

```diff
@@ -29,7 +30,8 @@ jobs:
           path: ~/.cache/ci/r128gain
           key: ${{runner.os}}-ffmpeg-${{hashFiles('ci/build-minimal-ffmpeg')}}-${{matrix.ffmpeg-version}}
       - run: |
-          sudo apt-get install yasm
+          sudo apt-get update -q
+          sudo apt-get install -qy yasm sox
```

Comment:
`-y` is not needed, I think because it runs with `DEBIAN_FRONTEND=noninteractive`.

Reply:
Good point, I have updated the commit to just add the `apt-get update` and the `-q` flags.